uwblueprint/childrens-aid-society
# Created familial and child concerns model

## Notion ticket link

[Create familial_concerns and child_concerns model](https://www.notion.so/uwblueprintexecs/134e1f159caa4f73939d11a700c325ad?v=39cf7d5a98384eb9af2bff2265d0ac8a&p=32d31a95f8c74163862ac4858ba0e377)



## Implementation description
* Created the model for familial_concerns and child_concerns tables
* Provided seed values to database
* Ran migration and upgraded the database



## Steps to test
1. Access the database - docker exec -it cas_py_backend /bin/bash
2. Run the command - SELECT * FROM child_concerns; or  SELECT * FROM familial_concerns;
3. If you see the table with the correct seed values, then the change should have been made.


## Checklist
